src/api/app/controllers/project/public_key_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Project
```

Comment:
In this particular case it is a class because we already have the `Project` model which is a class. Though I will be changing this to `module Projects` (plural) anyway so this will become a module..